app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="@dimen/spacing_large"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:src="@drawable/wolox_logo_large"/>
+
+    <EditText
+        android:id="@+id/fragment_login_email"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/spacing_large_extra"
+        android:layout_marginLeft="@dimen/spacing_large_extra"
+        android:layout_marginRight="@dimen/spacing_large_extra"
+        android:gravity="center"
+        android:hint="@string/fragment_login_email" />
+
+    <EditText
+        android:id="@+id/fragment_login_password"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/spacing_large_extra"
+        android:layout_marginLeft="@dimen/spacing_large_extra"
+        android:layout_marginRight="@dimen/spacing_large_extra"
+        android:gravity="center"
+        android:inputType="textPassword"
+        android:hint="@string/fragment_login_password" />
+
+    <Button
+        android:id="@+id/fragment_login_login"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/spacing_medium"
+        android:text="@string/fragment_login_login" />
+
+    <Button
+        android:id="@+id/fragment_login_signup"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/spacing_medium"
+        android:text="@string/fragment_login_signup" />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/spacing_none"
```

Comment:
You can use the value "0" here. It's one exception to use hardcode values , like layout_weight="1".